fitbenchmarking/utils/options.py

```diff
@@ -45,7 +45,8 @@ def __init__(self, file_name=None):
         plotting = config['PLOTTING']
         self.colour_scale = plotting.getlist('colour_scale')
         self.colour_scale = [(float(cs.split(',', 1)[0].strip()),
```

Comment:
This might look nicer as a for loop now that there's so many splits

Reply:
Since the mantid ranking is no longer used I have removed it from the options